src/wasm-binary.h

```diff
@@ -846,6 +847,10 @@ enum BulkMemoryOpcodes {
   MemoryFill = 0x0b
 };
 
+enum ExceptionHandlingOpcodes {
```

Comment:
Should this be part of a different PR? It seems unrelated to the feature.

Reply:
Done